client/src/pages/workspace/board/board.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {useState} from "react";
+import {IconButton} from '@mui/material';
+import {DeleteBoardProp} from "./props";
+import "./style.css";
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import MutableInput from '../mutableInput/MutableInput.component';
+
+const Board: React.FC<DeleteBoardProp> = ({board, deleteBoard}: DeleteBoardProp) => {
+    const [description, descriptionSet] = useState(board.description);
+    const [name, nameSet] = useState(board.name);
+
+    return (
+        <div className={"board"}>
```

Comment:
If you start using grid component from material ui https://mui.com/material-ui/react-grid/ or flexbox https://developer.mozilla.org/ru/docs/Learn/CSS/CSS_layout/Flexbox
 https://mui.com/system/flexbox/ your design will be more responsive